docs/language/README.md

```diff
@@ -211,10 +216,13 @@ using `switch`:
 from ... | switch color (
   "red" => op1 | op2 | ... ;
   "blue" => op1 | op2 | ... ;
-  default => op1 | op2 | ...
+  default => op1 | op2 | ... ;
```

Comment:
FYI, in addition to this one fix that brings this closer to the changes made in #2871, one of my goals in a future pass is to create a set of ztests that effectively fill in the `...` portions of these examples with real operations on real input files. While these will likely all be redundant to existing ztests, since I respect the desire to keep these abbreviated examples in the intro text (as opposed to the bulkier in-line mdtest-protected examples we use elsewhere), I'd like to have some tests close to these docs that are prominently labeled along the lines of **"If you find you need to adjust this test in order to make the CI pass, that means you need to update one of the abbreviated docs examples too!"**